pyproject.toml

```diff
@@ -9,11 +9,14 @@ requires = [
 build-backend = "setuptools.build_meta"
 
 [tool.pytest.ini_options]
-addopts = "--junitxml=junit/test-results.xml -n auto --strict-markers --cov-config=setup.cfg --cov=econml --cov-report=xml"
+# addopts = "--junitxml=junit/test-results.xml -n auto --strict-markers --cov-config=setup.cfg --cov=econml --cov-report=xml"
```

Comment:
Why is this being commented out? As a result, test results don't show up in the Azure Pipelines results, and code coverage results are not being generated and published.

Reply:
Ah, I just wanted to test running a single test and I wasn't able to do it with those options.